src/coreclr/tests/src/profiler/native/eventpipeprofiler/eventpipeprofiler.cpp

```diff
@@ -40,21 +40,21 @@ HRESULT EventPipeProfiler::Initialize(IUnknown* pICorProfilerInfoUnk)
 
     // Create a param descriptor for every type
     COR_PRF_EVENTPIPE_PARAM_DESC allTypesParams[] = {
-        { COR_PRF_EVENTPIPE_BOOLEAN, WCHAR("Boolean") },
-        { COR_PRF_EVENTPIPE_CHAR, WCHAR("Char") },
-        { COR_PRF_EVENTPIPE_SBYTE, WCHAR("SByte") },
-        { COR_PRF_EVENTPIPE_BYTE, WCHAR("Byte") },
-        { COR_PRF_EVENTPIPE_INT16, WCHAR("Int16") },
-        { COR_PRF_EVENTPIPE_UINT16, WCHAR("UInt16") },
-        { COR_PRF_EVENTPIPE_INT32, WCHAR("Int32") },
-        { COR_PRF_EVENTPIPE_UINT32, WCHAR("UInt32") },
-        { COR_PRF_EVENTPIPE_INT64, WCHAR("Int64") },
-        { COR_PRF_EVENTPIPE_UINT64, WCHAR("UInt64") },
-        { COR_PRF_EVENTPIPE_SINGLE, WCHAR("Single") },
-        { COR_PRF_EVENTPIPE_DOUBLE, WCHAR("Double") },
-        { COR_PRF_EVENTPIPE_GUID, WCHAR("Guid") },
-        { COR_PRF_EVENTPIPE_STRING, WCHAR("String") },
-        { COR_PRF_EVENTPIPE_DATETIME, WCHAR("DateTime") }
+        { 0, COR_PRF_EVENTPIPE_BOOLEAN,  WCHAR("Boolean") },
```

Comment:
```suggestion
        { COR_PRF_EVENTPIPE_BOOLEAN, 0, WCHAR("Boolean") },
```
I think the field order is swapped, typecode is first